dist/vis.js

```diff
@@ -7226,7 +7226,7 @@ return /******/ (function(modules) { // webpackBootstrap
 
       //! moment.js
 
-      hooks.version = '2.29.4';
+      hooks.version = '2.29.3';
```

Comment:
Hi, 

I am not sure about this hook. I have not changed this code, this might have got changed when I run the build command.
I am using node version 8.11.1 for the build.
Is it something that I need to change ?